library-extensions-expandable/src/main/java/com/mikepenz/fastadapter/utils/SubItemUtil.kt

```diff
@@ -256,30 +254,27 @@ object SubItemUtil {
             if (parent != null) {
                 parentPos = fastAdapter.getPosition(parent)
                 val subItems = parent.subItems
+                subItems.remove(item)
+                // check if parent is expanded and notify the adapter about the removed item, if necessary (only if parent is visible)
+                if (parentPos != -1 && parent.isExpanded) {
+                    expandableExtension.notifyAdapterSubItemsChanged(parentPos, subItems.size + 1)
+                }
 
-                subItems.let { subItems ->
```

Comment:
Basically just removed this line since it doesn't do anything. Change block is just indents